Load bossrush packages

steven11sjf opened the pull request:
- bumped MEDS to 0.33.0 (version checking)
- replaced deprecated logging functions
- consolidated ScenarioHelpers into functions on the enum
- updated RomManager to use the new ExtractedRomFs class
- added buttons for each bossrush package to Load Scenario menu
- scenarios are now enabled or disabled depending on if the scenario exists in the romfs version

steven11sjf committed:
Load bossrush packages

- bumped MEDS to 0.33.0 (version checking)
- replaced deprecated logging functions
- consolidated ScenarioHelpers into functions on the enum
- updated RomManager to use the new ExtractedRomFs class
- added buttons for each bossrush package to Load Scenario menu
- scenarios are now enabled or disabled depending on if the scenario
  exists in the romfs version

MayberryZoom commented on src/dreaditor/main_window.py:
```diff
@@ -84,6 +84,7 @@ def addPaintMenuAction(text: str, config_name: str):
         addPaintMenuAction("Logic Paths", "paintLogicPaths")
         addPaintMenuAction("World Graph", "paintWorldGraph")
 
+        self.updateMenuForRomVersion()
```
Wouldn't it be better to just not add Boss Rush maps to begin with for 1.0.0? This looks kinda weird. It could definitely look better with improved styling, but it would still be better to just not have them here imo.
![image](https://github.com/user-attachments/assets/61b836b0-5bae-4cd0-be94-559ea8781ce8)

steven11sjf replied:
good call, decided to make only "loadable" packages appear here. also prevents showing unusable options when a romfs is selected.

MayberryZoom commented on src/dreaditor/constants.py:
```diff
@@ -1,34 +1,52 @@
+    GHAVORAN = "s050_forest", "Ghavoran"
+    HANUBIA = "s080_shipyard", "Hanubia"
+    ITORASH = "s090_skybase", "Itorash"
+    BOSSRUSH_CORPIUS = "s201_bossrush_scorpius", "Corpius Bossrush", [GameVersion.DREAD_2_1_0]
```
Shouldn't it be "Boss Rush" (2 words)?

steven11sjf committed:
fixes

- fixed logging to actually print info to log files
- refactored scenario enum again
- load scenario menu only has functional buttons
